spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -32,35 +32,35 @@ Only this Specification will be generated and maintained to document the content
 
 In reverse chronological order, here's the specification for each MicroProfile release.
 
-[[microprofile4.0-M1]]
-=== MicroProfile 4.0 Milestone 1 (Q32020)
-// MicroProfile 4.0-M1 is the 14th platform release for the
-// https://projects.eclipse.org/projects/technology.microprofile[Eclipse MicroProfile project].
-MicroProfile 4.0-M1 is based on Jakarta EE 8, the first MicroProfile release to
+[[microprofile4.0-RC1]]
+=== MicroProfile 4.0 Release Candidate 1 (Q32020)
+// MicroProfile 4.0-RC1 is the 14th platform release for the
+// https://projects.eclipse.org/projects/technology.microprofile[MicroProfile project].
+MicroProfile 4.0-RC1 is based on Jakarta EE 8, the first MicroProfile release to
 be based on Jakarta EE (replacing the role of Java EE).
 Although Jakarta EE 8 is API backwards compatible with Java EE 8, Jakarta EE replaces Java EE dependencies with
 Jakarta EE dependencies for all MicroProfile specifications.
 
 The following specifications include API incompatible changes:
 
-* https://download.eclipse.org/microprofile/microprofile-config-2.0-M1/microprofile-config-spec.html#_incompatible_changes[Config]
-* https://download.eclipse.org/microprofile/microprofile-fault-tolerance-3.0-RC1/microprofile-fault-tolerance-spec.html#_backward_incompatible_changes[Fault Tolerance]
-* https://download.eclipse.org/microprofile/microprofile-health-3.0-RC3/microprofile-health-spec.html#release_notes_3_0[Health]
-* https://download.eclipse.org/microprofile/microprofile-metrics-3.0-RC1/microprofile-metrics-spec-3.0-RC1.html#_breaking_changes[Metrics]
-* https://download.eclipse.org/microprofile/microprofile-open-api-2.0-RC2/microprofile-openapi-spec.html#_release_notes_for_microprofile_openapi_2_0[Open API]
+* https://download.eclipse.org/microprofile/microprofile-config-2.0-RC1/microprofile-config-spec.html#_incompatible_changes[Config]
+* https://download.eclipse.org/microprofile/microprofile-fault-tolerance-3.0-RC2/microprofile-fault-tolerance-spec.html#_backward_incompatible_changes[Fault Tolerance]
+* https://download.eclipse.org/microprofile/microprofile-health-3.0-RC5/microprofile-health-spec.html#release_notes_3_0[Health]
+* https://download.eclipse.org/microprofile/microprofile-metrics-3.0-RC2/microprofile-metrics-spec-3.0-RC2.html#_breaking_changes[Metrics]
+* https://download.eclipse.org/microprofile/microprofile-open-api-2.0-RC3/microprofile-openapi-spec.html#_release_notes_for_microprofile_openapi_2_0[Open API]
```

Comment:
https://download.eclipse.org/microprofile/microprofile-open-api-2.0-RC3/microprofile-openapi-spec.html#_incompatible_changes[Open API]

Reply:
Updated this link in the document